security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,24 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-5-0"]
+== OpenShift Compliance Operator 1.5.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.5.0:
+
+* link:https://access.redhat.com/errata/RHBA-2024:3533[RHBA-2024:3533 - OpenShift Compliance Operator 1.5.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-5-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* As of this release, the Compliance Operator now provides a unique profile ID for easier programmatic use.  (link:https://issues.redhat.com/browse/CMP-2450[*CMP-2450*])
+
```

Comment:
Maybe we can add the ROSA hcp support as a new feature, too.

Reply:
@xiaojiey I'm confused - this is marked "private" on the JIRA but it definitely needs to be put in the release notes. I'll go ahead with this, because what you ask makes the most logical result!